pyproject.toml

```diff
@@ -43,6 +43,8 @@ addopts = "-rsxX --showlocals --tb=native -p no:cacheprovider --import-mode=impo
 # Transform all warnings into errors
 filterwarnings = [
     "error",
+    # Falcon currently uses cgi which is going away in python 3.13
+    "ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning",
```

Comment:
Falcon is planning to fix this in version 4 https://github.com/falconry/falcon/issues/2066, but it supports python 3.11 as of [Falcon 3.1.1](https://github.com/falconry/falcon/releases/tag/3.1.1)

For now python 3.13 isn't coming out for another 11 months, and @willkg is investigating switching to another micro-framework such as flask or fastapi

Reply:
Hi @relud, yes, we will release either 4.0 or another microrelease to make the framework work on the latest versions of CPython and PyPy.
I understand the wish to consolidate around a smaller number of frameworks, but in the case you reconsider, we are always open to feedback wrt "some pitfalls along the way" :wink: